### ISSUE TITLE
Relation to RFC6906 incorrect in abstract

### ISSUE THREAD

**Issue by RubenVerborgh** [larsgsvensson/I-D-Accept--Schema#7]
The abstract mentions that

> [This document] defines and registers the profile parameter for the HTTP Link header

but this is already done by RFC6906, as confirmed later in the text.

**Comment by larsgsvensson** [comment]
RFC 6906 defines the profile *relation* (`rel="profile"`). (This document) defines the keyword `profile` so that we can write `Link: rel="self"; profile="<urn:example:something>"`. There are some examples of that in the examples section. Obviously we need some more text to get this sorted out and I expected that to be part of the W3C rec we're to write, but probably we need some more text in the RFC, too.

**Comment by RubenVerborgh** [comment]
I'm not convinced about the necessity of such a new keyword instead of a relationship. Also see #10.

**Comment by larsgsvensson** [comment]
My point is that with the "profile" keyword we can supply more metadata about alternate resources, e. g. 
```
Link: rel="self";type="text/turtle";profile="<urn:example:profile-1">,
    <http://example.org/something-1.rdf>; rel="alternate"; type="application/rdf+xml"; profile="<urn:example:profile-1>",
    <http://example.org/something-2.rdf>; rel="alternate"; type="application/rdf+xml"; profile="<urn:example:profile-2>",
    <http://example.org/something-2.ttl"; rel="alternate"; type="text/turtle"; profile="<urn:example:profile-2>
```
I don't know how to do that with RFC 6906

**Comment by RubenVerborgh** [comment]
```
Link: <urn:example:profile-1>; rel=profile; type="type="text/turtle",
      <http://example.org/something-1.rdf>; rel="alternate"; type="application/rdf+xml",
      <http://example.org/something-2.rdf>; rel="alternate"; type="application/rdf+xml",
      <http://example.org/something-2.ttl"; rel="alternate"; type="text/turtle",
      <urn:example:profile-1>; rel=profile; anchor=<http://example.org/something-1.rdf>,
      <urn:example:profile-2>; rel=profile; anchor=<http://example.org/something-2.rdf>,
      <urn:example:profile-2>; rel=profile; anchor=<http://example.org/something-2.ttl>
```

**Comment by larsgsvensson** [comment]
OK, thanks. Not as straightforward but reuses existing solutions. Do you have a feeling which solution is easier to implement (for the consuming application)?

**Comment by RubenVerborgh** [comment]
The custom `;profile=` seems easier of course, but the question is whether the difference is worthwhile breaking with the `rel=` paradigm. (The good news is that RFC 5988 Web Linking allows such extensions.) Quick analysis below.

### Reusing `rel=profile`
#### Pro
- reuse of RFC6906, so existing clients remain compatible
- works like regular `rel=…` things
#### Con
- specifying both media type and profile on an alternate representation requires an extra `Link`

### Creating `profile= `
#### Pro
- single `Link` to specify both media type and profile on alternate representations
#### Con
- not reusing RFC6906

(Incomplete, but this already shows things depend on usage of RFC6906.)

**Comment by hvdsomp** [comment]
We need to have a look at:

https://tools.ietf.org/html/draft-nottingham-link-hint-00

1. I am not sure what the current plan with this I-D is
2. I had been in touch a while ago with Mark Nottingham re inclusion of a "profile" link hint. I actually even refer to it in http://signposting.org/conventions/#attributes I don't really remember whether Mark ever reacted.

Still, I think we should keep this in mind. Or at least check with Mark what the plan with the I-D is and whether "profile" could be supported.

Cheers

Herbert

**Comment by hvdsomp** [comment]
Check this tweet thread with Nottingham and Wilde re "profile" and negotiation:

https://mobile.twitter.com/evertp/status/854424773182595072

Herbert

**Comment by larsgsvensson** [comment]
Yes, very good questions and good perspectives. A conference call to collect the different views, pros and cons would a good path forward.
